pkg/consensus/consensus.go

```diff
@@ -440,11 +442,21 @@ func (s *service) verifyBlock(b block.Block) bool {
 		return false
 	}
 
+	size := io.GetVarSize(coreb)
```

Comment:
Serializing whole block can be expensive.

Reply:
I added `(b *Block) GetExpectedBlockSize()`, so now only transactions are being serialized. But transactions serialisation can also be expensive, so do we need the same helper for a transaction?

Reply:
We cache tx size.